web/themes/custom/pineapple/sass/components/_card.scss

```diff
@@ -84,3 +83,29 @@
     }
   }
 }
+
+.horizontal-teaser-container {
```

Comment:
This styling does not belong to this file. This file is for the styling of the individual card.

Reply:
Done.